src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets

```diff
@@ -1,14 +0,0 @@
-<!--
-***********************************************************************************************
-Microsoft.NET.Sdk.Razor.targets
-
-WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
-          created a backup copy.  Incorrect changes to this file will make it
-          impossible to load or build your projects from the command-line or the IDE.
-
-Copyright (c) .NET Foundation. All rights reserved.
-***********************************************************************************************
--->
-<Project ToolsVersion="14.0">
```

Comment:
Sorry what's the implication of deleting this file?

Reply:
We no longer import targets or props files from the files NuGet auto-imports. All they do is redirect where the Sdk looks for them. The Sdk will eventually import them.

The specific scenario that removing these files \ imports from the props file that it affects is that if if you reference Microsoft.NET.Razor.Sdk package in a project that does not reference `Microsoft.NET.Razor.Sdk` the Sdk, you no longer get any of the Razor features.